README.md

```diff
@@ -1,26 +1,30 @@
+[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/silverstripe-labs/silverstripe-crontask/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/silverstripe-labs/silverstripe-crontask/?branch=master)
+[![Scrutinizer Code Coverage](https://scrutinizer-ci.com/g/silverstripe-labs/silverstripe-crontask/badges/coverage.png?b=master)](https://scrutinizer-ci.com/g/silverstripe-labs/silverstripe-crontask/?branch=module-standard)
+[![Travis Build Status](https://travis-ci.org/silverstripe-labs/silverstripe-crontask.svg?branch=master)](https://travis-ci.org/silverstripe-labs/silverstripe-crontask)
```

Comment:
Can you change the Scrutinzer Code coverage link to `https://scrutinizer-ci.com/g/silverstripe-labs/silverstripe-crontask/?branch=module` and change the travis link to: `https://travis-ci.org/silverstripe/silverstripe-crontask`

Thanks!